doc/cookbook/splitting.md

```diff
@@ -66,3 +66,258 @@ $ pachctl put-file users master -c -f user_data.json --split json --target-file-
 # bytes chunk into the split files.
 $ pachctl put-file users master -c -f user_data.txt --split line --target-file-bytes 100
 ```  
+
+## Specifying Header/Footer
```

Comment:
It should be mentioned up front that there are put-header and put-footer commands.

Reply:
Ok added